modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -0,0 +1,2289 @@
+  params.addParam<MooseEnum>("energy_face_interpolation",
+                             face_interpol_types,
+                             "The numerical scheme to interpolate the temperature/energy to the "
+                             "face (separate from the advected quantity interpolation).");
+  params.addParam<MooseEnum>(
+      "passive_scalar_face_interpolation",
+      face_interpol_types,
+      "The numerical scheme to interpolate the passive scalar field variable to the "
+      "face (separate from the advected quantity interpolation).");
+
+  params.addParam<bool>(
+      "pressure_two_term_bc_expansion",
+      true,
+      "If a two-term Taylor expansion is needed for the determination of the boundary values"
+      "of the pressure.");
+  params.addParam<bool>(
+      "momentum_two_term_bc_expansion",
+      true,
+      "If a two-term Taylor expansion is needed for the determination of the boundary values"
+      "of the velocity/momentum.");
+  params.addParam<bool>(
+      "energy_two_term_bc_expansion",
+      true,
+      "If a two-term Taylor expansion is needed for the determination of the boundary values"
+      "of the temperature/energy.");
+  params.addParam<bool>(
+      "passive_scalar_two_term_bc_expansion",
+      true,
+      "If a two-term Taylor expansion is needed for the determination of the boundary values"
+      "of the advected passive scalar field.");
+
+  params.addRangeCheckedParam<Real>(
+      "mass_scaling",
+      1.0,
+      "mass_scaling > 0.0",
+      "The scaling factor for the mass variables (for incompressible simulation "
+      "this is pressure scaling).");
+  params.addRangeCheckedParam<Real>("momentum_scaling",
+                                    1.0,
+                                    "momentum_scaling > 0.0",
+                                    "The scaling factor for the momentum variables.");
+  params.addRangeCheckedParam<Real>("energy_scaling",
+                                    1.0,
+                                    "energy_scaling > 0.0",
+                                    "The scaling factor for the energy variables.");
+  params.addRangeCheckedParam<Real>("passive_scalar_scaling",
+                                    1.0,
+                                    "passive_scalar_scaling > 0.0",
+                                    "The scaling factor for the passive scalar field variables.");
+
+  params.addParamNamesToGroup(
+      "momentum_advection_interpolation energy_advection_interpolation "
+      "passive_scalar_advection_interpolation mass_advection_interpolation "
+      "momentum_face_interpolation energy_face_interpolation passive_scalar_face_interpolation "
+      "pressure_face_interpolation momentum_two_term_bc_expansion "
+      "energy_two_term_bc_expansion passive_scalar_two_term_bc_expansion "
+      "pressure_two_term_bc_expansion momentum_scaling energy_scaling "
+      "mass_scaling passive_scalar_scaling",
+      "Numerical control");
```

Comment:
```suggestion
      "Numerical scheme parameters");
```
as you prefer, this is personal preference

Reply:
I'd separate scaling from face interpolation parameters too

Reply:
Addressed!